step-functions/audit_event.asl.json

```diff
@@ -211,8 +211,34 @@
       "Comment": "This check if the input Details has an evidence to add to the audit structure",
       "Choices": [
         {
-          "Variable": "$$.Execution.Input.detail.evidence",
-          "IsPresent": true,
+          "And": [
+            {
+              "Variable": "$$.Execution.Input.detail.evidence",
+              "IsPresent": true
+            },
+            {
+              "Or": [
+                {
+                  "Variable": "$$.Execution.Input.detail.evidence[0].type",
+                  "IsPresent": true
+                },
+                {
+                  "Variable": "$$.Execution.Input.detail.evidence[0].ci",
+                  "IsPresent": true
+                },
+                {
+                  "Variable": "$$.Execution.Input.detail.evidence[0].ciReasons",
+                  "IsPresent": true
+                },
+                {
+                  "Not": {
+                    "Variable": "$$.Execution.Input.detail.evidence[0].txn",
+                    "StringEquals": ""
+                  }
```

Comment:
Why not just?

```
              "Or": [
                {
                  "Not": {
                    "Variable": "$$.Execution.Input.detail.evidence[0].txn",
                    "StringEquals": ""
                  }
```

Reply:
I think there is a `attemptNum` field `record_check` journey which is currently not in scope. In any case I believe the above is more flexible

Reply:
Hmm I put the other checks in incase the txn was empty on a VC_ISSUED so we did not ignore the rest of the evidence, but I suppose there should always be a txn if a VC has been issued. 
I'll remove the other checks 👍